fdbserver/VersionedBTree.actor.cpp

```diff
@@ -6193,7 +6414,7 @@ class VersionedBTree {
 				Reference<ArenaPage> page = self->m_pager->newPageBuffer();
 				makeEmptyRoot(page);
 				self->m_pHeader->height = 1;
-				self->m_pager->updatePage(newRootID, page);
+				self->m_pager->updatePage(PagerEventReasons::commit, rootLevel, newRootID, page);
```

Comment:
While `rootLevel` which is == 1 happens to be correct here, it's only because the tree was just cleared, the `rootLevel` constant is still wrong :)
The height of the tree is `self->m_pHeader->height` which was set to 1 on the previous line.